src/transaction/types.ts

```diff
@@ -1,4 +1,4 @@
-export const enum TxBodyKeys {
+export enum TxBodyKeys {
```

Comment:
just out of curiosity, why did you have to remove the `const` ?

Reply:
With `const` it isn't working here https://github.com/vacuumlabs/cardano-hw-cli/blob/upgrade-types-for-allegra-mary/src/transaction/guards.ts#L154